### PR TITLE
 python310Packages.finvizfinance: init at 0.14.5 

### PR DESCRIPTION
**Pull request by icyrockcom**
###### Description of changes

New package https://github.com/lit26/finvizfinance

finvizfinance is a package which collects financial information from FinViz website

PR for required datetime package: https://github.com/NixOS/nixpkgs/pull/215393.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by icyrockcom** [comment]
@natsukium If you're able / when you get a chance, can you CR this?

**Comment by mweinelt** [comment]
> Not sure who's the right person here, pinging @mweinelt since i saw you commit some python-related things recently... Let me know if I should ping someone else if you know.

Not interested in investing time into anything related to cryptocoins.

**Comment by icyrockcom** [comment]
> Not interested in investing time into anything related to cryptocoins.

Thanks for clarifying. Note this is a generic financing data fetching library and while there's apparently support for crypto it's not crypto specific. E.g. [the readme](https://github.com/lit26/finvizfinance/#quote) has a lot of focus on common stocks.

File: pkgs/development/python-modules/finvizfinance/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    substituteInPlace setup.py \
+      --replace "bs4" "beautifulsoup4"
+  '';
+
+  propagatedBuildInputs = [
+    beautifulsoup4
+    datetime
+    lxml
+    pandas
+    pytest
```
[the remark this answers is not in the document]
Comment (icyrockcom) [reply pkgs/development/python-modules/finvizfinance/default.nix]: Thanks, I've updated that - many tests require network access, so I've disabled a bunch. Let me know if that looks good.

File: pkgs/development/python-modules/finvizfinance/default.nix [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    requests
+  ];
+
+  pythonImportsCheck = [
+    "finvizfinance"
+  ];
+
+  meta = with lib; {
+    description = "Finviz Finance information downloader";
+    homepage = "https://github.com/lit26/finvizfinance";
```
[the remark this answers is not in the document]
Comment (icyrockcom) [reply pkgs/development/python-modules/finvizfinance/default.nix:65]: Apologizes, I've missed this - updated now. Let me know how it looks.

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 215395` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.finvizfinance</li>
    <li>python310Packages.finvizfinance.dist</li>
    <li>python311Packages.finvizfinance</li>
    <li>python311Packages.finvizfinance.dist</li>
  </ul>
</details>

**Review by natsukium — APPROVED** [review]
LGTM

**Comment by SuperSandro2000** [comment]
@ofborg build python310Packages.finvizfinance

**Pull request merged**